src/main/java/com/epam/reportportal/annotations/ReportPortalParameter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-api
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.epam.reportportal.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used in parametrized tests. Report portal waiting test parameters
+ * in <b>key - value</b> style. Using the annotation for method parameter
+ * the specific <b>key</b> can be provided.
+ *
+ * @author Pavel Bortnik
+ * @since Report Portal Api v3.1.0
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.PARAMETER })
+public @interface ReportPortalParameter {
```

Comment:
let's rename it to smth shorter. Can it be just ParameterName? Any ideas?

Reply:
ParameterKey ?